com.unity.multiplayer.mlapi/Documentation~/MigrateFromMLAPI.md

```diff
@@ -0,0 +1,31 @@
+# Migrate from MLAPI
```

Comment:
this is the outline I'm imagining so far.
please let me know what you guys think before we fully commit to this doc and write down all details in the sections below.

Reply:
_(and yes, we can change the outline/sections)_